api/env.template

```diff
@@ -56,4 +56,4 @@ IS_PROXIED=False
 
 FILTER_DEAD_LINKS_BY_DEFAULT=False
 ENABLE_FILTERED_INDEX_QUERIES=True
-SHOW_COLLECTION_DOCS=False
+SHOW_COLLECTION_DOCS=True
```

Comment:
Since it's the same value as the default would be, I would recommend commenting it out.

```suggestion
# SHOW_COLLECTION_DOCS=True
```